specification.md

```diff
@@ -1855,6 +1863,20 @@ for this connection.
 This field is optional: if omitted the client SHOULD NOT use a client-side certificate.
 This field can be used to perform a client certificate revocation/rotation.
 
+##### OpAMPConnectionSettings.heartbeat_interval_seconds
+
+The Client should use the offered heartbeat interval to periodically send an AgentToServer
+message. At a minimum the instance_uid field MUST be set. It is recommended that the Agent
+also set ComponentHealth as well. An HTTP based-client MUST use the heartbeat interval as
+its polling interval.
+
+A heartbeat is used to keep a load balancer connection active and inform the server that
+the Agent is still alive and active. A server could use the heartbeat to make decisions about
+the liveness of the connected Agent.
+
+A default of a 30s should be used if not set by the OpAMPConnectionSettings. If the
```

Comment:
A default of 30s second polling for WebSockets seems like an overkill. What was the typical timeout you were observing?

Reply:
I've seen timeouts after 30s running locally and in cloud environments. Per [these AWS docs](https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-websocket-api-integration-requests.html) 30s appears to be their default.

Reply:
[GCP's is 5 mins ](https://cloud.google.com/run/docs/triggering/websockets#client-reconnects)

Reply:
[Azure recommends a 20-40 second timeout](https://learn.microsoft.com/en-us/azure/application-gateway/application-gateway-websocket#backendaddresspool-backendhttpsetting-and-routing-rule-configuration)